src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -2144,7 +2144,11 @@ namespace HeatBalanceSurfaceManager {
 
             // Initialize previous-timestep shading indicators
             e.ExtIntShadePrevTS = 0;
-            e.ShadingFlag = NoShade;
+            // Not sure why is shade reset necessary here for rest of window models. BSDF window model has already
```

Comment:
Are you sure it **is** necessary?

Reply:
I have to avoid shading flag reset here. Will take a look tomorrow if I can move my initialization somewhere else.